content/200-concepts/200-database-connectors/07-mongodb.mdx

```diff
@@ -188,3 +188,36 @@ To resolve this, we suggest you change your deployment to one with a replica set
 One simple way for this is to use [MongoDB Atlas](https://www.mongodb.com/cloud/atlas) to launch a free instance that has replica set support out of the box.
 
 There's also an option to run the replica set locally with this guide: https://docs.mongodb.com/manual/tutorial/convert-standalone-to-replica-set
+
+## Type mapping between MongoDB and the Prisma schema
+
+> Alternatively, see the [Prisma schema reference](/reference/api-reference/prisma-schema-reference#model-field-scalar-types) for type mappings organized by Prisma type.
+
+The MongoDB connector maps the [scalar types](/concepts/components/prisma-schema/data-model#scalar-fields) from the Prisma [data model](/concepts/components/prisma-schema/data-model) to MongoDB's native column types as follows:
+
+| Prisma    | MongoDB      |
+| --------- | ------------ |
+| `String`  | `String`     |
+| `Bool`    | `Boolean`    |
+| `Double`  | `Double`     |
+| `Int32`   | `Int32`      |
+| `Int64`   | `Int64`      |
+| `Decimal` | `Decimal128` |
+| `Date`    | `DateTime`   |
+| `BinData` | `Binary`     |
+
+When introspecting a MongoDB database, the database types are mapped to Prisma according to the following table:
+
+| MongoDB (Type \| Aliases) | Prisma     | Supported | Native database type attribute | Notes |
+| ------------------------- | ---------- | :-------: | :----------------------------- | :---- |
+| `ObjectId`                | `ObjectId` |    ✔️     | `@db.ObjectId`                 |       |
```

Comment:
I don't know how accurate this is, please get a review from a schema team member on this.

Reply:
I think the `Prisma` column should say `String`, in the sense that the field would look like this:

```prisma
model Test {
  id String @db.ObjectId @map("_id") @default(auto()) @id
}
```

Reply:
```suggestion
| `ObjectId`                | `String` |    ✔️     | `@db.ObjectId`                 |       |
```

Reply:
Is `ObjectId` the correct capitalization from the MongoDB side @dpetrick?